checks/checks.go

```diff
@@ -244,23 +244,23 @@ func getFileOwner(info os.FileInfo) (uid, gid string) {
 }
 
 //Helper function to check rules in auditctl
-func checkAuditRule(rule string) bool {
+func checkAuditRule(rule string) (b bool, err error) {
```

Comment:
The error is raised when auditd doesn't exist or when the user doesn't have enough privileges to access the rules. The reason that every check has this failing condition is that we can't know which checks will be used in the first place.